.gitmodules

```diff
@@ -10,3 +10,6 @@
 	fxtag = v1.1.1
 	fxrequired = AlwaysRequired
 	fxDONOTUSEurl = https://github.com/jmaerz/M4AGO-sinking-scheme
+[submodule "pkgs/GSW-Fortran"]
+	path = pkgs/GSW-Fortran
+	url = https://github.com/TEOS-10/GSW-Fortran
```

Comment:
I am not entirely sure, but I would expect that git fleximod also supports the sha number of e.g. the master so that the behavior of the TEOS can at least be expected to be fixed. Maybe worth giving it a try.